docs/4.0/admin-guide.md

```diff
@@ -31,6 +31,26 @@ $ make full
 $ sudo mkdir -p /var/lib/teleport
 ```
 
+### Teleport Checksum
+
+Gravitational Teleport provides a checksum from the Downloads page.  This can be used to 
+verify the integrity of our binary. 
+
+![Teleport Checksum](img/teleport-sha.png)
+
+```bash
+$ shasum -a 256 teleport-v4.0.8-darwin-amd64-bin.tar.gz
```

Comment:
FYI `shasum -a 256` is a MacOS command. The Linux equivalent is `sha256sum` - I don't know whether we ought to have different examples here?

Reply:
Thanks Gus, that's helpful. I've improved my documentation.